macros/commands/get_tables_without_model.sql

```diff
@@ -0,0 +1,59 @@
+{% macro get_tables_without_models(deprecated_models_path=none) %}
+  {% set model_schemas, model_tables = elementary.get_model_schemas_and_tables(deprecated_models_path) %}
+  {% set db_tables = elementary.get_db_tables_in_schemas(model_schemas) %}
+  {% for db_table in db_tables %}
+    {% if db_table not in model_tables %}
+      {% do print(db_table) %}
+    {% endif %}
+  {% endfor %}
+{% endmacro %}
+
+
+{% macro get_db_tables_in_schemas(schemas) %}
```

Comment:
What do you think about calling it `get_tables_in_schemas`?
The `db` confused me a bit 😅